docs/src/sections/mockup.adoc

```diff
@@ -0,0 +1,108 @@
+administrator można stwierdzić po ikonie na górnym pasku nawigacji oraz po o wiele większej ilości ikon w panelu
+bocznym. Sam panel zarządzania administratorami składa się tabeli wyświetlającej szczegóły o administratorach, w tym
+informację o przypisaniu do portfela oraz o statusie przesłania klucza publicznego. W górnej części znajduje się
+przycisk przekierowujący do formularza pozwalającego na dodanie nowych administratorów.
+
+.Makieta ekrany zarządzania administratorami
+image::../images/mockup/admin_management.png[{image-size}]
+
+Aby uzyskać dostęp do portfela i wszystkich funkcjonalności z nim związanych należy go najpierw utworzyć z
+wykorzystaniem specjalnego formularza. Wyświetlać się on będzie w miejscu portfela, gdy nie będzie on zainicjowany.
+Formularz składa się z pola liczbowego, określającego wymaganą liczbę podpisów do odblokowania środków na adresie
+multisig. Drugim polem będzie lista rozwijana, która pozwoli na wybranie z listy administratorów z wygenerowanymi
+kluczami. Po wysłanie formularza zostanie utworzony portfel, a użytkownik zostanie przekierowany na jego ekran.
+
+.Makieta strony z ustawieniami serwera
+image::../images/mockup/wallet_setup.png[{image-size}]
+
+Ekran portfela składa się z kilku mniejszych komponentów wyświetlających informacje o stanie finansów serwera.
+Pierwszym komponentem jest stan portfela Bitcoin. Wyświetlana jest aktualnie posiadana kwota oraz kwota
+z niepotwierdzonych transakcji. Niepotwierdzona transakcja to taka, która została już wykopana przez kopacza,
+aczkolwiek nie zostało do danego bloku dołączonych następnych bloków. Zazwyczaj wymagane są 3 dołączone bloki, aby
+uznać transakcję za potwierdzoną, niemniej jednak jest to zależne od implementacji węzła bądź portfela.
+Drugim komponentem jest aktualny stan portfela znajdującego się w sieci Lightning. Tak samo, jak w przypadku
+poprzedniego komponentu, wyświetlane są środki znajdujące się w portfelu, jednakowo potwierdzone oraz niepotwierdzone.
+Ostatni komponent z pierwszego rzędu zawiera informację o środkach zablokowanych w kanałach płatności po stronie
+serwera. Drugą informacją jest suma aktualnie otwartych kanałów płatności. W dolnej części panelu znajduje się
+wykres, na których będzie można wyświetlić różne dane statystyczne związane z serwerem. Dla przykładu został podany
+całkowity dochód serwera ze sprzedaży tokenów. Ostatnim elementem znajdującym się w panelu portfela jest komponent
+akcji. Zawiera on przyciski, za pomocą których można wykonywać akcje w portfelu, takie jak zamknięcie kanałów czy też
+odtworzenie portfela.
+
+.Makieta ekranu portfela
+image::../images/mockup/wallet.png[{image-size}]
+
+Ostatnim z głównych części aplikacji mobilnej jest ekran transakcji. Podobnie jak ekran zarządzania administratorami,
+zawierać będzie listę transakcji, z wyszczególnieniem transakcji aktualnie trwającej (planowany jest limit jednej
```

Comment:
"zawierać on będzie"/ będzie on zawierał
nie wiem czy w nawiasie warto kolejny raz powtarzać transakcja

Reply:
jak wywale to nawias straci trochę kontekst

Reply:
racja, zostawmy tak